lib/stm32/h7/rcc.c

```diff
@@ -0,0 +1,26 @@
+/** @defgroup rcc_file RCC peripheral API
+ *
+ * @ingroup peripheral_apis
+ * This library supports the Reset and Clock Control System in the STM32 series
+ * of ARM Cortex Microcontrollers by ST Microelectronics.
+ *
+ * LGPL License Terms @ref lgpl_license
+ */
+
+#include <libopencm3/cm3/assert.h>
+#include <libopencm3/stm32/rcc.h>
+#include <libopencm3/stm32/pwr.h>
+#include <libopencm3/stm32/flash.h>
+
+/**@{*/
+
+uint32_t rcc_ahb_frequency = 64000000;
+uint32_t rcc_apb1_frequency = 64000000;
+uint32_t rcc_apb2_frequency = 64000000;
+
+// All PLL configurations without PLLM. PLLM should be set to the input clock
+// frequency in MHz.
+const struct rcc_clock_scale rcc_3v3[] = {
```

Comment:
was there meant to be something here?

Reply:
There are no clock configurations yet. Have removed this and the associated comment.